10up/distributor
# Fix a PHP fatal error with the `log_sync` function

### Description of the Change

In the latest Distributor release, we added an optional parameter to the `log_sync` method that is used to overwrite the log. The base method we extended had a type declaration on that optional value to `bool` but the two places we extended this method didn't have that same type declaration. In PHP versions 7.0 and 7.1, this will cause a fatal error to be thrown because those function signatures don't match.

In addition, PHP 5.6 does not support scalar type declarations, which `bool` is considered a scalar type. Because we still support PHP 5.6, this PR removes the `bool` type declaration so no errors will happen in 5.6, 7.0 or 7.1.

### Alternate Designs

None

### Benefits

No PHP fatals on PHP version <7.2

### Possible Drawbacks

None

### Verification Process

- Install the latest version of Distributor on a site running PHP <7.2
- Should get a fatal error
- Checkout this PR in same environment
- Everything should work now

### Checklist:


### Applicable Issues

Fixes #734. Related #735
